pkg/cloudvolume/logout.go

```diff
@@ -0,0 +1,65 @@
+// (C) Copyright 2019 Hewlett Packard Enterprise Development LP.
+
+package cloudvolume
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+)
+
+func newLogoutCommand() *cobra.Command {
+	var host string
+
+	var cmd = &cobra.Command{
+		Use:   "logout",
+		Short: "Logout from HPE Cloud Volumes",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			if host != "" && !strings.HasPrefix(host, "http") {
+				host = fmt.Sprintf("https://%s", host)
+			}
+
+			if host == "" {
+				host = cvDefaultHost
+			}
+
+			return runLogout(host)
+		},
+	}
+
+
```

Comment:
You need to read the flags or it will always be empty.  similar to the login command.
cmd.Flags().StringVar(&opts.host, "host", "", "HPE Cloud Volumes portal hostname/ip")

Reply:
I don't get this comment as for logout I have no flags left